### ISSUE TITLE
How to access request object inside `AddCustomRule` function?

### ISSUE THREAD

**Issue by thedevsaddam** [thedevsaddam/govalidator#26]
Probably we'll pass the request object to `AddCustomRule` to next major release (v.20) , but to do this we need more discussion

**Comment by ramadani** [comment]
yup, this is an ability we need to create a custom rule like `confirmed` rule on laravel validation rules. so the `confirmed` rule needs to access the value from another field value.
